### PR TITLE
Minor corrections: Grafana & Prometheus

### PR DESCRIPTION
**Pull request by mbogoevici**

### CODE REVIEW

File: rh-summit-2018/module-01.adoc [hunk 1]
```diff
@@ -251,15 +251,12 @@ To deploy Grafana on OpenShift, the following commands should be executed:
 $ oc create -f https://raw.githubusercontent.com/strimzi/strimzi/master/metrics/examples/grafana/kubernetes.yaml
 
 For accessing the dashboard, you can use the `port-forward` command for forwarding traffic from the Grafana pod to the host.
-For example, you can access the Grafana UI by running :
+For example, you can access the Grafana UI after running:
 
 [source,sh]
-$ oc port-forward grafana-1-fbl7s 3000:3000
+$ oc expose svc/grafana
```
Comment (ppatierno) [inline rh-summit-2018/module-01.adoc]: now that we are exposing the Grafana dashboard through a Route, should we say where the attendee can find the link to that Route for opening the UI ?

File: rh-summit-2018/module-01.adoc [hunk 2]
```diff
@@ -270,12 +267,15 @@ Click on the "Add data source" button from the Grafana home in order to add Prom
 image::grafana_home.png[grafana home]
 
 Fill in the information about the Prometheus data source, specifying a name and "Prometheus" as type.
-In the URL field, the connection string to the Prometheus server `http://prometheus:9090`` should be specified.
+In the URL field, use `http://prometheus.l1099-kafka.svc.cluster.local:9090` as the URL to the Prometheus server.
```
Comment (ppatierno) [inline rh-summit-2018/module-01.adoc]: Isn't it enough specifying just the service name ("prometheus") and not the complete name with namespace ? Did you try and it didn't work ?

File: rh-summit-2018/module-01.adoc (hunk 1 above)
Comment (mbogoevici) [reply rh-summit-2018/module-01.adoc]: We should point them to the Routes tab in the UI, or just how to get it from the CLI.

File: rh-summit-2018/module-01.adoc (hunk 2 above)
Comment (mbogoevici) [reply rh-summit-2018/module-01.adoc]: Yeah, both are correct. Either way works for me.

File: rh-summit-2018/module-01.adoc (hunk 1 above)
Comment (mbogoevici) [reply rh-summit-2018/module-01.adoc]: As a side note, we should do the same in module 2 (and wherever we're having routes).

**Pull request merged**